airflow/jobs.py

```diff
@@ -1829,7 +1845,7 @@ def _execute(self):
             run.verify_integrity(session=session)
 
             # check if we have orphaned tasks
```

Comment:
Can change the comment to "Clean up any orphaned tasks" if you want while you're at it (or just remove it, I think the function name is pretty self explanatory).